src/opera_utils/_utils.py

```diff
@@ -153,3 +157,94 @@ def scratch_directory(
 
     if delete and not dir_already_existed:
         shutil.rmtree(scratchdir)
+
+
+def get_snwe(epsg: int, bounds: Bbox) -> Bbox:
+    """Convert bounds from (west, south, east, north) (WSEN) to SNWE in lat/lon.
+
+    This box format is used by the `RAiDER` library for its area of interest tracking:
+    https://github.com/dbekaert/RAiDER/blob/38eab3969ac762bc59502d7eb482fd73d6a0deef/tools/RAiDER/llreader.py#L30
+
+    Parameters
+    ----------
+    epsg : int
+        EPSG code.
+    bounds : tuple[float, float, float, float]
+        Bounds in WSEN format.
+
+    Returns
+    -------
+    tuple[float, float, float, float]
+        Bounds in SNWE (lat/lon) format.
+    """
+    if epsg != 4326:
+        snwe = reproject_bounds(bounds, epsg, 4326)
+
+    else:
+        snwe = (bounds[1], bounds[3], bounds[0], bounds[2])
+
+    return snwe
+
+
+def create_yx_arrays(
+    gt: list[float], shape: tuple[int, int], step_size: float = 500
+) -> tuple[np.ndarray, np.ndarray]:
+    """Create the x and y coordinate datasets.
+
```

Comment:
```suggestion
    """Create the x and y coordinate datasets.

    Assumes that the `y` output coordinates will be north-up, so that the 
    `y` array is in decreasing order.
```

(I wanted to add this note because I realized that this output has the `y` flipped from what I was doing in `disp-s1`. It now makes me want to check if NetCDF products make assumptions about which order the `x/y` coordinates will be in, or if I was actually causing our products to be flipped...)